### DISCUSSION TITLE
Don't track bots?

### DISCUSSION THREAD

**Discussion by youradds** [plausible/analytics#1859] [Self-Hosted Support] [answered]
Hi,

Is there a way to NOT track when a bot is on the site? Or should it be doing this by default? I have had a look on the discussions, and find some mention of bots, but nothing on how to exclude them? 

Thanks

Andy

**Comment by metmarkosaric** [comment, marked as answer]
hi! all the known non-human traffic is excluded by default. see https://plausible.io/docs/dashboard-faq#does-plausible-exclude-known-bots-and-spam-traffic

**Reply by youradds** [reply]
Awesome, thanks for the quick reply! I'm so used to crappy AWS where they don't do that ;)
